src/main/scala/beam/physsim/jdeqsim/JDEQSimRunner.scala

```diff
@@ -41,7 +41,7 @@ class JDEQSimRunner(
 ) extends StrictLogging {
 
   def simulate(currentPhysSimIter: Int, writeEvents: Boolean): SimulationResult = {
-    val jdeqsimEvents = new EventsManagerImpl
+    val jdeqsimEvents = new ParallelEventsManagerImpl(10)
```

Comment:
I'm assuming that 10 comes from the total number of event handlers, correct?
- LegHistogram
- PhyssimCalcLinkStats
- EventToHourFrequency
- EventTypeCounter
- CarTravelTimeHandler
- TravelTimeCalculator
- JDEQSimMemoryFootprint
- PhyssimSpeedHandler
- PhysSimEventWriter (optional)

Reply:
That's correct